homeassistant/components/ambiclimate/__init__.py

```diff
@@ -19,6 +21,8 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+PLATFORMS = [Platform.CLIMATE]
```

Comment:
I'd personally see this more in `const.py` of each specific integration, but not sure. Let's wait for some more feedback.

Reply:
Not sure that's necessary. It's only used in the init files after all. Furthermore for existing integrations, it's about evenly split maybe with a slight edge towards init compared to const.

Reply:
Please keep constants that are only used in a single module in that module.